extensions/amp-list/0.1/test/test-amp-list.js

```diff
@@ -30,16 +30,21 @@ describes.realWin('amp-list component', {
   let element;
   let list;
   let listMock;
+  let viewerMock;
   let setBindService;
 
   beforeEach(() => {
     win = env.win;
     doc = win.document;
     ampdoc = env.ampdoc;
+    ampdoc.isSingleDoc = () => true;
```

Comment:
I think you can pass `'single'` to the describes initializer object, e.g.

https://github.com/ampproject/amphtml/blob/25ba76cfb5cefe16a65344c80f5617fdeb9a7260/extensions/amp-user-notification/0.1/test/test-amp-user-notification.js#L30-L35

Would that work?

Reply:
yes, thanks!